pygsti/construction/modelconstruction.py

```diff
@@ -1254,52 +1253,58 @@ def create_crosstalk_free_model(num_qubits, gate_names, nonstd_gate_unitaries={}
+                                     depolarization_parameterization, stochastic_parameterization,
+                                     lindblad_parameterization)
+        # TODO: Could do qubit-specific by allowing different err_gate1Q
+        err_gates = [err_gate1Q.copy() for i in range(num_qubits)] if independent_gates else [err_gate1Q,] * num_qubits
+        err_gateNQ = _op.ComposedOp([
+            _op.EmbeddedOp(range(num_qubits), [i], err_gates[i]) for i in range(num_qubits)])
+        # TODO: Could specialize for LindbladOps, although this should also handle that case
+        prep_layers['rho0'] = ComposedSPAMVec(rho_baseNQ, err_gateNQ, 'prep')
     else:
         prep_layers['rho0'] = _spamvec.ComputationalSPAMVec([0] * num_qubits, evotype, 'prep')
 
     povm_layers = {}
     if 'povm' in all_keys:
-        if 'povm' in depolarization_strengths and depolarization_parameterization != 'lindblad':
-            _warnings.warn(("'povm' error specification requires Lindblad parameterization, "
-                           "depolarization parameterization '%s' overridden!" % depolarization_parameterization))
-        elif 'povm' in stochastic_error_probs and stochastic_parameterization != 'lindblad':
-            _warnings.warn(("'povm' error specification requires Lindblad parameterization, "
-                           "stochastic parameterization '%s' overridden!" % stochastic_parameterization))
-
-        Mdefault_base1Q = _povm.ComputationalBasisPOVM(1, evotype)
-        # Override parameterization to force Lindblad
-        err_gate = _get_error_gate('povm', _op.StaticStandardOp('Gi', evotype), depolarization_strengths,
-                                   stochastic_error_probs, lindblad_error_coeffs,
-                                   "lindblad", "lindblad", "auto")
-        povm1Q = _povm.LindbladPOVM(err_gate, Mdefault_base1Q, "pp")
-        povm_factors = [povm1Q.copy() for i in range(num_qubits)] if independent_gates else [povm1Q] * num_qubits
-        povm_layers['Mdefault'] = _povm.TensorProdPOVM(povm_factors)
+        # Mdefault_base1Q = _povm.ComputationalBasisPOVM(1, evotype)
+        # err_gate = _get_error_gate('povm', _op.StaticStandardOp('Gi', evotype), depolarization_strengths,
+        #                            stochastic_error_probs, lindblad_error_coeffs,
+        #                            depolarization_parameterization, stochastic_parameterization,
+        #                            lindblad_parameterization)
+        # if isinstance(err_gate, _op.LindbladOp):
+        #     povm1Q = _povm.LindbladPOVM(err_gate, Mdefault_base1Q, "pp")
+        # else:
+        #     povm1Q = ComposedPOVM(err_gate, Mdefault_base1Q)
+        # povm_factors = [povm1Q.copy() for i in range(num_qubits)] if independent_gates else [povm1Q] * num_qubits
+        # povm_layers['Mdefault'] = _povm.TensorProdPOVM(povm_factors)
+        Mdefault_baseNQ = _povm.ComputationalBasisPOVM(num_qubits, evotype)
```

Comment:
Same comment as above - should there be an option to create a TensorProdPOVM?

Reply:
Same solution as above.